eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -215,7 +215,7 @@ jobs:
       Windows_Release_MapFiles:
         OSVmImage: 'windows-2019'
         vcpkg.deps: ''
-        VCPKG_DEFAULT_TRIPLET: 'x64-windows-static'
+        VCPKG_DEFAULT_TRIPLET: 'x64-windows-static-md'
```

Comment:
Does it make sense to use `-md` even for the release leg?

Reply:
It is not. `-md` stands for "multithreaded dynamic".

Reply:
Consider updating references `x64-windows-static` in docs to include `-md`. We have a couple of places if you search for `triplet`:
https://github.com/Azure/azure-sdk-for-c/blob/ef22522414e5b56acba056eae84dd3facda63865/sdk/samples/iot/README.md#set-environment-variables

https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/samples/iot/docs/how_to_iot_hub_samples_windows.md#setup-instructions